models/responses.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+import (
+	"time"
+
+	"github.com/skycoin/dmsg/buildinfo"
+)
+
+// HealthcheckResponse is structe of /health endpoint
+type HealthcheckResponse struct {
+	BuildInfo            *buildinfo.Info `json:"build_info,omitempty"`
+	NumberOfClients      int64           `json:"clients,omitempty"`
```

Comment:
What do you think of removing the `omitempty` attribute here and on the line below? With it, if a client or server count is 0, they are not shown in the output, but I think it would be good to have them.